docker-compose.dev.yml

```diff
@@ -0,0 +1,15 @@
+version: "2"
+services:
+  app:
+    build:
+      context: .
+      dockerfile: build/Dockerfile.dev
+    image: go-api-template
```

Comment:
Why?

Reply:
If we don't need an image with default repo name and tag then sure I will drop it